lib/hooks/core/hook-http.js

```diff
@@ -101,8 +101,6 @@ function uriFromOptions(options) {
   var uri;
   if (typeof options === 'string') {
     uri = options;
-  } else if (options.uri && options.uri.href) {
```

Comment:
`npm.im/request` is probably the reason this existed. See also: https://github.com/request/request/blob/master/request.js#L204

Looking at _http_client though, even if `uri` made it to here, it would be the wrong thing for us to use. The following would have logged the incorrect url: `http.get({uri: "http://wrong.com/bar", host: "google.com" }...`, so thanks for fixing.